python/svix/api/exceptions.py

```diff
@@ -0,0 +1,11 @@
+from ..openapi_client.exceptions import (  # noqa
```

Comment:
Why not expose all of the exceptions from `svix/exceptions.py`? Both API and receiver?